operators/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags="$GO_TAGS" -a -o proces
 
 # Copy the controller-manager into a thin image
 FROM ubuntu:latest
-WORKDIR /root/
+RUN mkdir /eck && chmod 755 /eck
+WORKDIR /eck/
```

Comment:
Another way would be to put the binaries in the existing `/usr/local/bin`, then just use: 
`WORKDIR /usr/local/bin`. It will save us one layer :).

Reply:
If a user `elastic`  is added to the image we can use its home directory `/eck`
IIRC it would be more or less consistent with the docker images in cloud where binaries are store in `/elastic_cloud_apps`. WDYT ?

Reply:
`/eck` looks good to me.